__tests__/__integration__/extension.integration.test.ts

```diff
@@ -603,78 +603,170 @@ describe("Extension Integration Tests", () => {
 
-                    try {
-                        const node = new ZoweDatasetNode(dataSetName, vscode.TreeItemCollapsibleState.None, sessionNode, session);
-                        node.contextValue = DS_DS_CONTEXT;
+            beforeEach(async () => {
+                await zowe.Delete.dataSet(sessionNode.getSession(), dataSetName).catch((err) => err);
+                await zowe.Create.dataSet(
+                        sessionNode.getSession(),
+                        zowe.CreateDataSetTypeEnum.DATA_SET_SEQUENTIAL,
+                        dataSetName,
+                    );
+            });
 
-                        await dsActions.hMigrateDataSet(node);
-                    } catch (err) {
-                        error = err;
-                    }
-                    expect(error).to.not.equal(undefined);
-                }).timeout(TIMEOUT);
+            afterEach(async () => {
+                await zowe.Delete.dataSet(sessionNode.getSession(), dataSetName).catch((err) => err);
             });
+
+            it("Should send a Recall request", async () => {
+                let error;
+
+                try {
+                    const node = new ZoweDatasetNode(dataSetName, vscode.TreeItemCollapsibleState.None, sessionNode, session);
+                    node.contextValue = DS_DS_CONTEXT;
+
+                    await dsActions.hRecallDataSet(node);
+                } catch (err) {
+                    error = err;
+                }
+                expect(error).to.be.equal(undefined);
+            }).timeout(TIMEOUT);
+        });
+        describe("Recall a partitioned data set", () => {
+            const dataSetName = `${pattern}.PDATA.REC`;
+
+            beforeEach(async () => {
+                await zowe.Delete.dataSet(sessionNode.getSession(), dataSetName).catch((err) => err);
+                await zowe.Create.dataSet(
+                        sessionNode.getSession(),
+                        zowe.CreateDataSetTypeEnum.DATA_SET_PARTITIONED,
+                        dataSetName,
+                    );
+            });
+
+            afterEach(async () => {
+                await zowe.Delete.dataSet(sessionNode.getSession(), dataSetName).catch((err) => err);
+            });
+
+            it("Should send a Recall request", async () => {
+                let error;
+
+                try {
+                    const node = new ZoweDatasetNode(dataSetName, vscode.TreeItemCollapsibleState.None, sessionNode, session);
+                    node.contextValue = DS_DS_CONTEXT;
+
+                    await dsActions.hRecallDataSet(node);
+                } catch (err) {
+                    error = err;
+                }
+                expect(error).to.be.equal(undefined);
+            }).timeout(TIMEOUT);
+        });
+    });
+    describe("Failure Scenarios", () => {
+        describe("Recall a sequential data set", () => {
+            const dataSetName = `${pattern}.TEST.FAIL`;
```

Comment:
I think Ive been recommended before to use a more random name like `pattern.test.A23HBG9` to ensure the ds doesnt exist

Reply:
@CForrest97 thanks for your comment, I have been told before that using `TEST.FAIL` is a good way to name for failure scenarios, but I would be happy to change this if you think that the way recommended to you is better